go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/aquasecurity/trivy-plugin-aqua
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2020-8911   
**Check Name:** aws/aws-sdk-go: CBC padding oracle issue in AWS S3 Crypto SDK for golang   
**Severity:** MEDIUM   
**Fixed Version:**    
**Description:** A padding oracle vulnerability exists in the AWS S3 Crypto SDK for GoLang versions prior to V2. The SDK allows users to encrypt files with AES-CBC without computing a Message Authentication Code (MAC), which then allows an attacker who has write access to the target's S3 bucket and can observe whether or not an endpoint with access to the key can decrypt a file, they can reconstruct the plaintext with (on average) 128*length (plaintext) queries to the endpoint, by exploiting CBC's ability to manipulate the bytes of the next block and PKCS5 padding errors. It is recommended to update your SDK to V2 or later, and re-encrypt your files.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2020-8911